navit/support/gettext_intl/localealias.c

```diff
@@ -88,6 +88,10 @@ char *alloca ();
 __libc_lock_define_initialized (static, lock);
 #endif
 
+#if defined(__MINGW32__) && !defined(__MINGW32CE__)
+# define HAVE_MEMPCPY   1
+#endif
+
```

Comment:
As the build in circleci now fails with
```
support/gettext_intl/libsupport_gettext_intl.a(dcigettext.c.obj): In function `libintl_dcigettext':
/root/project/navit/support/gettext_intl/dcigettext.c:575: undefined reference to `_mempcpy'
support/gettext_intl/libsupport_gettext_intl.a(localealias.c.obj): In function `read_alias_file':
/root/project/navit/support/gettext_intl/localealias.c:230: undefined reference to `_mempcpy'
collect2: ld returned 1 exit status
```
it looks as something is wrong with that condition now.

Reply:
ah ok.

Reply:
OK so far